### PR TITLE
Bugfix/#64: Replace old implementation to check empty regions with contrib module…

### PR DESCRIPTION
**Pull request by jclemmenb4y**
This pull request relates to [BOSA_0100-358](https://jira.hosted-tools.com/browse/BOSA_0100-358) (JIRA issue) and replaces old implementation to check empty regions with contrib module "Twig Real Content".

Module "[Twig Real Content](https://www.drupal.org/project/twig_real_content)" should also be added in OpenFed as declaring in Kiso's dependencies currently doesn't trigger the installation and doesn't enable the module (see https://www.drupal.org/node/2937955).

Function "has_region" declared in file "preprocess.utils.inc" might need to be kept currently to avoid issue with custom themes that override page.html.twig. What do you think @smillart ?

### CODE REVIEW

File: templates/layout/page.html.twig [hunk 1]
```diff
@@ -89,7 +85,7 @@
       {{ page.breadcrumb }}
     {% endblock %}
 
-    {% if has_navigation or has_complementary %}
+    {% if page.navigation|render is real_content %}
```
Comment (smillart) [inline templates/layout/page.html.twig:88]: Missing test if `complementary` region is empty:

```twig
{% if page.navigation|render is real_content or page.complementary|render is real_content %}
```

**Review by smillart — CHANGES_REQUESTED** [review]
Hey @jclemmenb4y,
Your changes should be re-based on the `3.0.x` (default) branch. The `3.1.x` branch is a "beta" candidate for a "possible" new future release of Kiso with Openfed Next.

**Comment by jclemmenb4y** [comment]
@smillart I've created another pull request based on branch 3.0.x, including the missing test for complementary region.
See https://github.com/openfed/kiso/pull/120
